eth/downloader/downloader.go

```diff
@@ -268,6 +269,7 @@ func (d *Downloader) Synchronise(id string, head common.Hash, td *big.Int, mode
 	default:
 		glog.V(logger.Warn).Infof("Synchronisation failed: %v", err)
```

Comment:
Expand the switch. There are 5 branches, and I'd need to add the return to all of them.